app/plugins/router.js

```diff
@@ -34,7 +34,8 @@ const routes = [].concat(
   require('../routes/withdraw'),
   require('../routes/withdrawn'),
   require('../routes/agreement'),
-  require('../routes/clear-session')
+  require('../routes/clear-session'),
+  require('../routes/delete')
```

Comment:
Maybe better to move into agreement folder in the routes

Reply:
Done